### ISSUE TITLE
Implement `Create Task` scenario

### ISSUE THREAD

**Issue by anxolerd** [pinhanderler/Googol#4]
> 1. User taps the “create new task” button
> 2. Application shows a new screen where user can enter task details, which include: title and optional long description with task details
> 3. System verifies if task details are correct (i.e. title is not blank)
> 4. If task details are correct, the system creates the task

- Operate with dummy storage for now
- Create a new activity for creating tasks
- Task details should not wipe when user changes device orientation 

Depends on: #2 

/assign @senaertugrul
